dev-project/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: python:3.7.4-buster
     container_name: pipelinewise_dev
     working_dir: /usr/src/app
-    entrypoint: /usr/src/app/entrypoint-dev.sh
+    entrypoint: /usr/src/app/dev-project/entrypoint.sh
     environment:
       PIPELINEWISE_HOME: /usr/src/app
```

Comment:
I think `PIPELINEWISE_HOME` has to be `/usr/src/app/dev-project` otherwise it will use `.virtualenvs` folder from root and not from `dev-project`.

Changing this might require some other changes in entrypoint-dev.sh

Reply:
will do it